CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.17.4 (2021-08-30)
```

Comment:
```suggestion
# 4.17.4 (2021-08-31)
```
Assuming this won't go out tonight, so adding a suggestion to update the date. I'll approve though in case you can get it out tonight or first thing in the morning.